docs/installing-oggm.rst

```diff
@@ -127,7 +127,11 @@ Packages
 
 Install the packages from the `conda-forge`_ channel::
 
-    conda install -c conda-forge geopandas matplotlib Pillow joblib netCDF4 scikit-image configobj nose pyproj numpy krb5 rasterio xarray filelock
+    conda install -c oggm -c conda-forge oggm-deps
+
+If you want to install oggm itself to just use it, install oggm instead of oggm-deps.
+oggm-deps is a meta package that contains no actual content, and just depends on all
+dependencies of oggm.
 
```

Comment:
I *think* I understand what you mean, but it could be clearer. Most people reaching this page are python newcomers, and need to be taken by the hand a little bit.

Do you mean that a ``conda install -c oggm -c conda-forge oggm`` will install all dependencies + oggm per git (i.e. not in dev mode)?